pay-api/src/pay_api/services/fas/routing_slip_status_transition_service.py

```diff
@@ -64,7 +64,9 @@ def get_possible_transitions(cls, rs_model: RoutingSlipModel, **kwargs) -> List[
         """Return all the status transition available."""
         transition_list: List[RoutingSlipStatus] = RoutingSlipStatusTransitionService.STATUS_TRANSITIONS.get(
             rs_model.status, [])
-        if RoutingSlipStatus.REFUND_AUTHORIZED.value in transition_list:
+        if RoutingSlipStatus.REFUND_AUTHORIZED.value in transition_list \
+                or \
+                RoutingSlipStatus.WRITE_OFF_AUTHORIZED.value in transition_list:
```

Comment:
any way you can combine it into one line? rather than an OR

Reply:
yes:)